adapter/pyproject.toml

```diff
@@ -46,6 +48,7 @@ bigquery = ["google-cloud-bigquery"]
 redshift = ["sqlalchemy-redshift"]
 duckdb = []
 athena = []
+teleport = ["s3fs"]
```

Comment:
Can we make a ticket to

```py
yield 'dbt-fal[<adapter>,teleport]', ...
```

when using teleport?

Reply:
We have a "test teleport" ticket, and I'm sure we will resolve this issue there.